### ISSUE TITLE
IMC 385 Pretrain

### ISSUE THREAD

**Issue by sowmen** [ZhiHanZ/IRIS0-SPAN#6]
Could you provide details on the IMC 385 pretraining? Optimizer, scheduler, amount of train data, did you perform kcmi testing?

**Comment by michiganleon** [comment]
Please check [ManTra-Net](https://ieeexplore.ieee.org/document/8953774) for the detail of the IMC 385 pre-training. We just directly use that model and freeze the weight in our training.
